core/requestlogger/domain/main.go

```diff
@@ -0,0 +1,28 @@
+package domain
+
+import (
+	"fmt"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+
+	"flamingo.me/flamingo/v3/framework/opencensus"
+)
+
+var (
+	// HTTPResponseCount counts different HTTP responses
+	HTTPResponseCount = stats.Int64("flamingo/requestlogger_http_response_count", "Count of http responses by status code", stats.UnitDimensionless)
+
+	// KeyHTTPStatus defines response http status code
+	KeyHTTPStatus, _ = tag.NewKey("status_code")
+
+	// KeyArea identifies the current application area
+	KeyArea, _ = tag.NewKey("area")
+)
+
+func init() {
+	if err := opencensus.View("flamingo/requestlogger_http_response_count", HTTPResponseCount, view.Count(), KeyHTTPStatus); err != nil {
```

Comment:
View holds valid values only if config `requestlogger.metrics.responseCountTracking.enabled` is `true`.  Should this also be handled to ensure valid metrics?

Reply:
The view will now only registered if the flag is true.